### ISSUE TITLE
payload (Ox6F) doesn't support this macOS version!

### ISSUE THREAD

**Issue by pyinto** [koekeishiya/yabai#2465]
On Yabai restart I always receive this notification, though scrip-addition seems to be working:
![screenshot 31-10-2024 at 15 49 47@2x](https://github.com/user-attachments/assets/87dc7b7f-e013-4f6a-a317-53a6be63f2ab)

I've tried to reinstall it from scratch - no changes.
System Integrity Protection is disabled, tho i didn't do anything after upgrading  from 15.0 to 15.1, should I redo any of [the steps described here](https://github.com/koekeishiya/yabai/wiki/Disabling-System-Integrity-Protection) after system upgrade? 
If not, what else could it be?

```
Privacy - Accessibility is granted for yabai;

$ yabai --version
yabai-v7.1.4

$ sw_vers
ProductName:            macOS
ProductVersion:         15.1
BuildVersion:           24B83

$ csrutil status
System Integrity Protection status: unknown (Custom Configuration).
```

**Comment by pyinto** [comment]
Not sure if it's related to https://github.com/koekeishiya/yabai/issues/1936 or similar

**Comment by koekeishiya** [comment]
Just need some minor adjustment for macOS Sequoia 15.1. I'll look into it this weekend and publish a new release.

**Comment by koekeishiya** [comment]
Fixed in v7.1.5.
